app/controllers/concerns/foreman_remote_execution/api/v2/hosts_controller_extensions.rb

```diff
@@ -0,0 +1,19 @@
+module Concerns::ForemanRemoteExecution::Api::V2::HostsControllerExtensions
+  extend Apipie::DSL::Concern
+
+  api :GET, '/hosts/:id/ssh_params', N_('Get default parameters for SSH to a host')
+  def ssh_params
+    render :json => SSHExecutionProvider.ssh_params(@host)
+  end
+
+  protected
+
+  def action_permission
+    case params[:action]
+    when 'ssh_params'
+      'view'
```

Comment:
I wonder if we should have some more expiclit persmission here, as one can get to ssh details though this end-point?

Reply:
Yes, that's a good question. Are there other ways already to piece together the information returned by this method?  For example, the password might also be available via /api/v2/hosts/:id as a parameter.

```
$ curl -s -b _session_id=9dda32ff969c0feed7c9ccb29ef62382 http://localhost:3000/api/v2/hosts/dev.mvo.lan | jq .parameters
[
  {
    "priority": 70,
    "created_at": "2018-11-29 10:10:37 UTC",
    "updated_at": "2018-11-29 10:10:37 UTC",
    "id": 1,
    "name": "remote_execution_ssh_password",
    "value": "foobar"
  }
]
```